# Fixes related to specific dlr-sc oce patch

diff --git src/SWIG_files/wrapper/GeomFill.i src/SWIG_files/wrapper/GeomFill.i
index bbb7c2b..32ad950 100644
--- src/SWIG_files/wrapper/GeomFill.i
+++ src/SWIG_files/wrapper/GeomFill.i
@@ -970,48 +970,6 @@ class GeomFill_BSplineCurves {
 	:rtype: None
 ") GeomFill_BSplineCurves;
 		 GeomFill_BSplineCurves (const Handle_Geom_BSplineCurve & C1,const Handle_Geom_BSplineCurve & C2,const GeomFill_FillingStyle Type);
-		%feature("compactdefaultargs") Init;
-		%feature("autodoc", "	* if the curves cannot be joined
-
-	:param C1:
-	:type C1: Handle_Geom_BSplineCurve &
-	:param C2:
-	:type C2: Handle_Geom_BSplineCurve &
-	:param C3:
-	:type C3: Handle_Geom_BSplineCurve &
-	:param C4:
-	:type C4: Handle_Geom_BSplineCurve &
-	:param Type:
-	:type Type: GeomFill_FillingStyle
-	:rtype: None
-") Init;
-		void Init (const Handle_Geom_BSplineCurve & C1,const Handle_Geom_BSplineCurve & C2,const Handle_Geom_BSplineCurve & C3,const Handle_Geom_BSplineCurve & C4,const GeomFill_FillingStyle Type);
-		%feature("compactdefaultargs") Init;
-		%feature("autodoc", "	* if the curves cannot be joined
-
-	:param C1:
-	:type C1: Handle_Geom_BSplineCurve &
-	:param C2:
-	:type C2: Handle_Geom_BSplineCurve &
-	:param C3:
-	:type C3: Handle_Geom_BSplineCurve &
-	:param Type:
-	:type Type: GeomFill_FillingStyle
-	:rtype: None
-") Init;
-		void Init (const Handle_Geom_BSplineCurve & C1,const Handle_Geom_BSplineCurve & C2,const Handle_Geom_BSplineCurve & C3,const GeomFill_FillingStyle Type);
-		%feature("compactdefaultargs") Init;
-		%feature("autodoc", "	* Initializes or reinitializes this algorithm with two, three, or four curves - C1, C2, C3, and C4 - and Type, one of the following filling styles: - GeomFill_Stretch - the style with the flattest patch - GeomFill_Coons - a rounded style of patch with less depth than that of Curved - GeomFill_Curved - the style with the most rounded patch. Exceptions Standard_ConstructionError if the curves are not contiguous.
-
-	:param C1:
-	:type C1: Handle_Geom_BSplineCurve &
-	:param C2:
-	:type C2: Handle_Geom_BSplineCurve &
-	:param Type:
-	:type Type: GeomFill_FillingStyle
-	:rtype: None
-") Init;
-		void Init (const Handle_Geom_BSplineCurve & C1,const Handle_Geom_BSplineCurve & C2,const GeomFill_FillingStyle Type);
 		%feature("compactdefaultargs") Surface;
 		%feature("autodoc", "	* Returns the BSpline surface Surface resulting from the computation performed by this algorithm.
 
-- 
